client/src/components/LibraryNavigator/LibraryModel.ts

```diff
@@ -23,6 +24,7 @@ class LibraryModel {
 
   public async connect(): Promise<void> {
     const session = getSession();
+    session.onLogFn = LogChannelFn;
```

Comment:
The `LogChannelFn` will always **show** the output channel.
Do user like the output channel popup if they're not running code?

Reply:
Confirmed that the UX of showing the log is acceptable.